modules/ROOT/pages/flex-gateway-modes.adoc

```diff
@@ -0,0 +1,25 @@
+= Flex Gateway Modes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Anypoint Flex Gateway can run in two modes:
```

Comment:
can run in > supports (assuming this is accurate and means the same)
or, > runs in 
(to avoid modal "can")

Reply:
Applied!